pybio/core/transformations/reshape.py

```diff
@@ -9,7 +11,7 @@ def __init__(self, shape: Sequence[int], **super_kwargs):
         self.shape = shape
```

Comment:
Do we have two `Reshape` transformations now? One from `numpylike` and the one here?

Reply:
yes, however there is only one spec (for the non-numpylike one)

Reply:
I guess we could just remove the numpylike Reshape, I left it in as its a reasonable numpylike one... (and allows for more [specific] kwargs)